src/appleseedmaya/renderglobalsnode.cpp

```diff
@@ -346,11 +459,11 @@ MStatus RenderGlobalsNode::initialize()
     numAttrFn.setMin(1);
     CHECKED_ADD_ATTRIBUTE(m_mbDeformSamples, "deformSamples")
 
-    // Shutter Open.
+    // Shutter open.
```

Comment:
I know that's completely unrelated to this PR, but probably worth mentioning anyway: appleseed now supports four times:

  - shutter open begin
  - shutter open end
  - shutter close begin
  - shutter close end

as well as various opening/closing curves for the [begin, end] intervals.